Description: remove extra newline in MIME structure
 MUA from MS will barf at the signature when wrapped in SMIME
Origin: https://rt.cpan.org/Public/Bug/Display.html?id=63253

--- a/lib/MIME/Lite.pm
+++ b/lib/MIME/Lite.pm
@@ -2179,7 +2179,7 @@
         }
 
         ### Epilogue:
-        $out->print("\n--$boundary--\n\n");
+        $out->print("\n--$boundary--\n");
     } elsif ( $type =~ m{^message/} ) {
         my @parts = @{ $self->{Parts} };
 
--- a/t/types.t
+++ b/t/types.t
@@ -65,7 +65,6 @@
 
 This isn't really html. We are only checking the filename silly.
 --_----------=_0--
-
 EOFEXPECT
 
 is($ret, $expect, "we got the message we expected");
